src/test/integration/scheduler/test_regression_virtual_file_path.py

```diff
@@ -28,32 +27,26 @@ def add_task(self, task):
         pass
 
 
-@pytest.fixture(scope='module')
```

Comment:
I'm not sure why these were module scope.

We don't really lose anything by having the [test_app](https://github.com/fkie-cad/FACT_core/pull/883/files#diff-41a46107cade9ba46cb2964e45cc140cdec08d97948ba3d51c14bb631cd6ff24R41-R44) fixture be function scoped.

It now hast to be function scoped because of the cfg_tuple fixture.

Reply:
I mean there is an obvious benefit: If the setup takes long, we can save a lot of time. But it's probably negligible here.

Reply:
>  If the setup takes long

Yeah the if is important here. Just instanciating a class should not take that long.